scripts/HPC_chatbot_preprocessor/parsed_mds/generic/HOD/HOD_paragraph_1.txt

```diff
@@ -0,0 +1,43 @@
+Hanythingondemand (HOD)
```

Comment:
@boegel  was this reently tested/used?

Reply:
No, we should remove this page. Pretty sure it's not working anymore, and there's no way we can maintain it.

=> https://github.com/hpcugent/vsc_user_docs/pull/693